draft-ietf-taps-interface.md

```diff
@@ -2272,21 +2290,25 @@ Name:
 : max-send-rate / max-recv-rate
 
 Type:
-: Integer (positive)
+: Integer (positive) / Integer (positive)
```

Comment:
-1 is not a positive integer :P

```suggestion
: Integer / Integer
```

Reply:
(On second thought, do we want to restrict this beyond Numeric?)

Reply:
Tsssssssssssk what a silly mistake!  Thanks!
Re: numeric, I absolutely agree!  I even thought about it as I made this PR and wanted to fix this (because it was an Integer before) but somehow forgot it.